### PR TITLE
Make double-click select words in line doc comments

### PR DESCRIPTION
**Pull request by stevemessick**
@alexander-doroshko Fixes https://youtrack.jetbrains.com/issue/WEB-17046

### CODE REVIEW

**Comment by alexander-doroshko** [comment]
I think it should be testable. There must be tests for other `ExtendWordSelectionHandler`'s to look at.

**Comment by stevemessick** [comment]
If you know where to put the test I'll be glad to add one. I don't know how to test double-click's in this system.

**Comment by alexander-doroshko** [comment]
Just a guess based on extension name: test similar to one of `*WordSelectionTest` could do the trick.

**Comment by alexander-doroshko** [comment]
Note that your extension affects not only double click but also incremental 'word selection' (Alt+Up / Alt+Down) - very handy IDE feature!

**Comment by stevemessick** [comment]
@alexander-doroshko PTAL I added tests for selecting words in all kinds of comments. Most of the tests are not in scope of this base change, but apparently they weren't being tested.

File: Dart/testSrc/com/jetbrains/lang/dart/ide/editor/DartWordSelectionTest.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import com.intellij.testFramework.fixtures.CodeInsightTestUtil;
+import com.jetbrains.lang.dart.DartCodeInsightFixtureTestCase;
+import com.jetbrains.lang.dart.util.DartTestUtils;
+
+public class DartWordSelectionTest extends DartCodeInsightFixtureTestCase {
+
+  @Override
+  public void setUp() throws Exception {
+    super.setUp();
+    DartTestUtils.configureDartSdk(myModule, getTestRootDisposable(), true);
```
Comment (alexander-doroshko) [inline Dart/testSrc/com/jetbrains/lang/dart/ide/editor/DartWordSelectionTest.java]: I'll change the last argument to `false`, we do not need real Dart SDK and real Analysis Server for these tests

**Comment by alexander-doroshko** [comment]
Pushed, thank you for the tests!

**Pull request merged**